src/libsync/configfile.h

```diff
@@ -42,6 +42,7 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
 
     enum Scope { UserScope,
         SystemScope };
+    bool visible;
```

Comment:
Data members should be private

Reply:
Yes, will move to private section with meaningful name "_isComponentVisible"